prow/config/tide.go

```diff
@@ -83,6 +83,22 @@ type TidePriority struct {
 type Tide struct {
```

Comment:
I think we also want `PrioritizeExistingBatchesMap` to be in the main `Tide` struct since that is not provider specific.

Reply:
done